tests/openQA_containers/multiple_container_webui.pm

```diff
@@ -0,0 +1,74 @@
+ use strict;
+ use base "openQAcoretest";
+ use testapi;
+ 
+ sub wait_for {
+     my ($container) = @_;
+     my $count = 30;
+     while (system("docker logs $container 2>&1 | grep Listening >/dev/null") != 0) {
+         sleep 1;
+         $count = $count - 1;
+         last if ($count == 0);
+     }
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L1485 does exactly this. In another way `wait_for` (i like the name) could be moved to the lib/utils.pm, so it could be reused here and there in the future. thoughts?

Reply:
Yes, I am working to do this more generic. I think I should create a container_utils instead of put that in utils.pm directly

Reply:
keep in mind there is already a folder in lib for containers with a utils.pm. Maybe you would like to unify them. Personally i like `container_utils` as a name

Reply:
@b10n1k you might be confusing https://github.com/os-autoinst/os-autoinst-distri-opensuse with the current repo https://github.com/os-autoinst/os-autoinst-distri-openQA

Reply:
Done, finally I put this in utils.pm (I found this more appropriate)